Pointwise.lua

```diff
@@ -69,5 +88,7 @@ end
 
 function Pointwise:clearState()
    self:clearDesc()
+   self._gradOutput = nil
+   nn.utils.clear(self, 'activDesc')
```

Comment:
Do you think _gradOutput requires clear() ? In other places it's just being set to nil.